### PR TITLE
LPS-147536 Add empty state

### PR DESCRIPTION
**Pull request by victorg1991**
Hello dear reviewer!

This PR remove the FF for the categories selector and also adapt it to work properly with the keyboard.

**After reviewing this don't forward it since we need to do a QA pass**

**You need to deploy the module** `asset-categories-item-selector-web`

More info here: https://issues.liferay.com/browse/LPS-153482

This category selector can be displayed in the following scenarios:
(Create some categories/vocabularies before testing)

**Adding a category to a navigation menu**
1. Add a navigation menu
2. Add a category

**Editing a category in a navigation menu**
1. Add a navigation menu
2. Add a category
3. Click on the category 
4. In the sidebar click the change button

**Selecting a category as preview item in a display page**
1. Add a display page of categories
2. Click preview with button
3. Choose category tab

@david-gutierrez-mesa @liulukiwi @Tim-Cao @ozysouza

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:relevant

File: modules/apps/asset/asset-categories-item-selector-web/src/main/resources/META-INF/resources/js/AssetCategoryTree.es.js [hunk 1]
```diff
@@ -63,12 +45,14 @@ export function AssetCategoryTree({
...
 		}
 
-		return performFilter(filterQuery.toLowerCase(), [...items]);
+		return nodeByName(items, filterQuery);
 	}, [items, filterQuery]);
 
 	const selectedItemsRef = useRef(new Map());
 
-	const handleMultipleSelectionChange = (selection, item) => {
+	const handleMultipleSelectionChange = (item, selection) => {
```
Comment (sandrodw3) [inline modules/apps/asset/asset-categories-item-selector-web/src/main/resources/META-INF/resources/js/AssetCategoryTree.es.js:54]: As we change params order here, I think we should change it also in the checkbox `onChange` calls, they are not working now 😄

**Review by sandrodw3 — COMMENTED** [review]
Super cool! 🥰  I just left a comment regarding the checkboxes.

File: modules/apps/asset/asset-categories-item-selector-web/src/main/resources/META-INF/resources/js/AssetCategoryTree.es.js (hunk 1 above)
Comment (victorg1991) [reply modules/apps/asset/asset-categories-item-selector-web/src/main/resources/META-INF/resources/js/AssetCategoryTree.es.js:54]: true :D I'll change that

**Comment by liulukiwi** [comment]
Hi @victorg1991 , I failed the test about Editing a category in a navigation menu, could you take a look at the [comment]( https://issues.liferay.com/browse/LPS-153482?focusedCommentId=2688601&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-2688601)? thanks!

**Comment by liulukiwi** [comment]
ci:test:bundle

**Comment by liulukiwi** [comment]
Hi @david-gutierrez-mesa , It looks good on Chrome, Firefox and Edge, could you check on Safari when you have time? thanks!

**Comment by liulukiwi** [comment]
Hi @david-gutierrez-mesa , could you take care of Safari? thanks!

**Comment by david-gutierrez-mesa** [comment]
Just started reviewing :)

**Comment by david-gutierrez-mesa** [comment]
Working fine in safari

**Comment by david-gutierrez-mesa** [comment]
ci:forward

**Comment by victorg1991** [comment]
ci:forward

**Pull request closed without merging**